sample/gutters.css.scss

```diff
@@ -0,0 +1,64 @@
+
+@import '../grid-system.scss';
+
+.app {
+	@include vui-gs-container;
+}
+
+.app * {
+	box-sizing: border-box;
+}
+
+.header {
+	@include vui-gs-row;
+	background-color: rgba(0,0,250,0.4);
+	height: 60px;
+}
+
+.stuff-1a {
+	@include vui-gs-span(4);
+	background-color: rgba(250,0,0,0.4);
+	height: 60px;
+}
+
+.stuff-1b {
+	@include vui-gs-span(4);
+	background-color: rgba(250,0,0,0.4);
+	height: 60px;
+}
+
+.stuff-1c {
+	@include vui-gs-span(last 4);
+	background-color: rgba(250,0,0,0.4);
+	height: 60px;
+}
+
+.stuff-2a {
+	@include vui-gs-span(4 wide no-gutter);
```

Comment:
I think they would be most common in this case:
```
@include vui-gs-span(last #);
@include vui-gs-span(1 of 8);
@include vui-gs-span(3 break);
```
In the first case, `last` indicates that it's the last span in a row, and so it will not include a gutter on the right side.

In the second case, the `of` keyword is used to change context, and is used when you want to effectively nest grids.  For example, let's say you have a span(6), and inside of that you want to break that down into 8 sections, the inner spans would be like `@include vui-gs-span(1 of 8)`.

In the third case, the `break` keyword can be used to break to a new row without having to create another `div` as a new row.

I think the other keywords would be used less commonly.